hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/Route.java

```diff
@@ -82,15 +86,27 @@ public void configure() {
                 .handled(true)
                 .id("ValidationException");
         
+        onCompletion().choice()
+        .when(header("isFileDropsAllowed")
+        		.isEqualTo("Y"))
+        .bean(V2FileDrops.class).id("V2FileDrops");
+
+        
         setupSSLConextPharmanetRegistry(getContext());
         String pharmNetUrl = String.format(pharmanetUri + "?bridgeEndpoint=true&sslContextParameters=#%s&authMethod=Basic&authUsername=%s&authPassword=%s", SSL_CONTEXT_PHARMANET, pharmanetUser, pharmanetPassword);
         String basicToken = buildBasicToken(pharmanetUser, pharmanetPassword);
         log.info("Using pharmNetUrl: " + pharmNetUrl);
+        
+			
+		String isFileDropsAllowed = properties.getValue(IS_FILEDDROPS_ALLOWED);
```

Comment:
Some small issues with the merge I think. In ApplicationProperty.java this is now IS_FILEDROPS_ENABLED